.changeset/red-rockets-joke.md

```diff
@@ -2,4 +2,4 @@
 "create-cloudflare": patch
 ---
 
-fix: cloning non-existent or private repository should fail gracefully
+fix: creating an application from a non-existent or private repository will no longer cause the CLI to get stuck in the cloning stage
```

Comment:
Thanks for updating! Can you also add what the expected behaviour now is?

"Now, when cloning a non-existent repo the CLI will exit with the error message '...' and when cloning a private repo you will be prompted for the username/password."

Reply:
done.